resources/js/components/sections/bookshelfAddStep3.vue

```diff
@@ -3,6 +3,12 @@
     <h3 class="text-2xl font-semibold mb-3"> {{ $t( 'bookshelfAdd-step3-title' ) }} </h3>
     <div class="grid grid-cols-1 md:grid-cols-2 gap-5 ml-8">
       <Button v-for="(Type, index) in transactionType" :key="Type.name" @click="selectType(index)" theme="square" :color="(selectedTypeIndex === index) ? 'primary' : ''">{{ Type.name }} </Button>
+      <div class="flex" v-if="selectedTypeIndex === 2">
+        <label>
+          {{ $t( 'bookshelfAdd-step3-price' ) }}
+        </label>
+        <input type="number" step="0.01" :value="price" class="BookshelfAddBookshelf-field rounded-md border-2 pl-8 pr-6 py-2 block mb-4 mt-1" @keyup="$emit('update:price', $event.target.value)" @change="$emit('update:price', $event.target.value)">
```

Comment:
You should be able to use `v-model` here.. so remove :value and @keyup and @change and just add

```<input v-model="price" plus all the classes and the rest />```

The above will automatically get the value and set the value

Reply:
I tried implementing this, but the price isn't updating as it should. I will investigate a bit later.